monitor/launch/launcher.go

```diff
@@ -29,37 +30,53 @@ const targetName = "cilium-node-monitor"
 
 // NodeMonitor is used to wrap the node executable binary.
 type NodeMonitor struct {
-	Arg     string
+	mutex   sync.RWMutex
+	arg     string
 	process *os.Process
 	state   *models.MonitorStatus
 }
 
 // Run starts the node monitor.
 func (nm *NodeMonitor) Run() {
 	for {
-		cmd := exec.Command(targetName, nm.Arg)
-		stdout, _ := cmd.StdoutPipe()
+		nm.mutex.RLock()
```

Comment:
Let's do these as `set` and `get` functions instead and do the locking inside those functions, that makes it obvious that it needs to be locked and it simplifies the code a lot.